lib/src/base-styles/_nx-global-header.scss

```diff
@@ -9,11 +9,12 @@
 
 .nx-global-header {
   align-items: center;
-  border-bottom: var(--nx-border-default);
+  background-color: #FFFFFF;
   display: flex;
   grid-area: header; // see nx-page's grid-template
-  min-height: 32px;
-  padding: var(--nx-spacing-6x);
+  margin-bottom: var(--nx-spacing-6x);
```

Comment:
This spacing is already implemented in the main content area and sidebar.  The problem with implementing it here like this is the scrollbars.
![image](https://user-images.githubusercontent.com/3630091/134978198-a9ac9cf3-c4b6-4a9a-b067-cdffa45933fc.png)

Reply:
> This spacing is already implemented in the main content area and sidebar. The problem with implementing it here like this is the scrollbars. ![image](https://user-images.githubusercontent.com/3630091/134978198-a9ac9cf3-c4b6-4a9a-b067-cdffa45933fc.png)

The extra spacing has been removed.
Check 65ed208c842eb774b0c4687a8f86793aa07abae4